examples/amp-list-with-form.amp.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <script async src="/dist/amp.js"></script>
+  <script async custom-element="amp-list" src="/dist/v0/amp-list-0.1.max.js"></script>
```

Comment:
I believe we are supposed to use the CDN URLs in our examples, and the `app.js` server replaces them automagically with the correct local or remote file depending on the serve_mode